includes/admin/class-sensei-onboarding.php

```diff
@@ -164,6 +173,7 @@ public function prepare_wizard_page() {
 		add_action( 'admin_print_scripts', [ $this, 'enqueue_scripts' ] );
 		add_action( 'admin_print_styles', [ $this, 'enqueue_styles' ] );
 		add_action( 'admin_body_class', [ $this, 'filter_body_class' ] );
+		add_action( 'init', [ $this, 'setup_wizard_set_script_translations' ] );
```

Comment:
`wp_set_script_translations` will only work after the enqueue/register call, so instead of the `init` hook this should be in `enqueue_scripts`.

Reply:
It was fixed here: https://github.com/Automattic/sensei/pull/3219/commits/e7c577e32c65ceb3330bd87e44e2892ece8d8289 ;)